sk0-back/src/users/services/users/users.service.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable } from '@nestjs/common';
+import { CreateUserDto } from 'src/users/dtos/createUser.dto';
+import { User } from 'src/users/types/User';
+
+@Injectable()
+export class UsersService {
```

Comment:
This one should be stateless as well, we can have a static method to provide mock data, I am not against it, but the service itself should be a wrapper around the Data Access Layer, with extra methods.

I know technically works, but is not a good practice. You introduce in-memory caching, which is hard to maintain.

https://docs.nestjs.com/fundamentals/injection-scopes <- about scopes

Reply:
The service should have an injected Repository (if we have one) rather.